docs/docusaurus/docs/components/connect_to_data/link_lists/_connecting_to_data_fluently.md

```diff
@@ -6,21 +6,21 @@
 </summary>
 
 **Local Filesystems**
-- [How to quickly connect to a single file using Pandas](/docs/guides/connecting_to_your_data/fluent/filesystem/connect_filesystem_source_data)
-- [How to connect to one or more files using Pandas](/docs/guides/connecting_to_your_data/fluent/filesystem/connect_filesystem_source_data)
-- [How to connect to one or more files using Spark](/docs/guides/connecting_to_your_data/fluent/filesystem/connect_filesystem_source_data)
+- [Connect to a single file using Pandas](/docs/guides/connecting_to_your_data/fluent/filesystem/connect_filesystem_source_data)
+- [Connect to one or more files using Pandas](/docs/guides/connecting_to_your_data/fluent/filesystem/connect_filesystem_source_data)
+- [Connect to one or more files using Spark](/docs/guides/connecting_to_your_data/fluent/filesystem/connect_filesystem_source_data)
 
 **Google Cloud Storage**
-- [How to connect to data on GCS using Pandas](/docs/guides/connecting_to_your_data/fluent/cloud/how_to_connect_to_data_on_gcs_using_pandas)
```

Comment:
These links can also probably be consolidated, since all of these pages are now tabs on a single page.

Reply:
@Rachel-Reverie I have consolidated the links.